api-docs/rst/dev-guide/concepts.rst

```diff
@@ -217,6 +254,51 @@ servers.
  .. _List Load Balancer Details: http://docs.rackspace.com/loadbalancers/api/v1.0/clb-getting-started/content/List_LB_Details.html
 
 
+.. _stack-parameters:
+
+Stack parameters
+^^^^^^^^^^^^^^^^
+
+Stack parameters specify the configuration of the stacks that will be
+created during the scaling process. Note the ``stack`` arguments are
+directly passed to heat when creating a stack. Note the stack arguments
+are directly passed to Heat when creating a stack. For more information,
```

Comment:
Looks like this information is duplicated: "Note the stack arguments
+are directly passed to Heat when creating a stack. " Remove the second occurrence (line 264, 265)